wbooks/src/app/screens/dashboard/screens/Home/components/BookList/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Book from './components/Book';
+import './styles.css';
+
+const BookList = (props) => {
+  return (
+    <div className="book-list">
+      {props.books.map((book) =>
+        <Book key={book.id} img={book.image_url} title={book.title} author={book.author} />
```

Comment:
Little tip:
A faster way to map object field to props is using `spread` operator:
```
<Book key={book.id} {...book} />
```
`{...book}` will pass all content to `Book` component, so the later one will receive `image_url` (not `img`), `title` and `author` as props.